graphql_client_codegen/src/generated_module.rs

```diff
@@ -51,7 +52,15 @@ impl<'a> GeneratedModule<'a> {
             CodegenMode::Derive => quote!(),
         };
 
+        let variables_type = match self.operation.variables.len() {
+            0 => quote!(()),
+            _ => quote!(
+            #module_name::Variables
```

Comment:
Might it not be easier to define it, but just as `type Variables = ();` so that the *name* also always exists?